app/main.py

```diff
@@ -4,4 +1,43 @@
+import os
+from typing import Dict
+
+import requests
+
+API_KEY = os.getenv("API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+if not API_KEY:
+    raise ValueError("API_KEY environment variable not set")
+
+
+def get_weather(city: str) -> Dict:
+    """Fetch current weather data for the specified city."""
+    params = {
+        "key": API_KEY,
+        "q": city,
+    }
+    response = requests.get(BASE_URL, params=params)
+    response.raise_for_status()
+    return response.json()
+
+
+def format_weather_output(data: Dict) -> str:
+    """Format the weather data into the desired string."""
+    location = data["location"]
+    current = data["current"]
+
+    city_name = location["name"]
+    country = location["country"]
+    local_time = location["localtime"]
+    temp_c = current["temp_c"]
+    condition = current["condition"]["text"]
```

Comment:
It's a good practice to check if the expected keys exist in the response data before accessing them. This can prevent KeyError exceptions if the API response format changes or if there is an issue with the data.